modules/appeals_api/spec/requests/v1/notice_of_disagreements_controller_spec.rb

```diff
@@ -69,9 +69,9 @@ def base_path(path)
       end
 
       it 'returns error objects in JSON API 1.0 ErrorObject format' do
-        expected_keys = %w[code detail links meta sentry_type source status title]
```

Comment:
links and sentry_type are optional keys and were causing failures as they were passing nil.